docs/get_started/installation.md

```diff
@@ -60,7 +60,7 @@ conda create --name camel python=3.10
 conda activate camel
 
 # Clone github repo
-git clone -b v0.2.6 https://github.com/camel-ai/camel.git
+git clone -b v0.2.7 https://github.com/camel-ai/camel.git
```

Comment:
I wonder how these doc files are updated. I hope that we don't need to manually update these.

Reply:
I searched for the old version number and then replaced it with the new version number in bulk. Do you know of a better method to accomplish this?